runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/RuntimeResponse.kt

```diff
@@ -0,0 +1,34 @@
+package io.openruntimes.kotlin
+
+import io.javalin.plugin.json.JsonMapper
+
+class RuntimeResponse(
+    var data: String = "",
+    var statusCode: Int = 200
+) {
+    fun json(data: Map<String, Any?>, statusCode: Int = 200): RuntimeResponse {
+        this.data = mapper?.toJsonString(data) ?: ""
+        this.statusCode = statusCode
+        return this
+    }
+
+    fun send(data: String, statusCode: Int = 200): RuntimeResponse {
+        this.data = data
+        this.statusCode = statusCode
+        return this
+    }
+
+    companion object {
+        var mapper: JsonMapper? = null
+
+        @JvmStatic
+        fun error(error: Throwable): RuntimeResponse {
+            return RuntimeResponse("{\"message\": \"" + error.message + "\"}", 500)
```

Comment:
The response here should be a simple string rather than a JSON string
https://github.com/open-runtimes/open-runtimes/blob/3261573d3a02e7dbad1868e465d2858a3a183494/runtimes/deno-1.13/server.ts#L45